bin/gobblin-mapreduce.sh

```diff
@@ -14,26 +14,47 @@ fi
 
 JOB_TRACKER_URL=$1
 FS_URL=$2
-JOB_CONFIG_FILE=$3
+JOB_CONFIG_FILE=$4
+
+# Convert job jar's relative path to absolute path
+DIRNAME=`dirname "$3"`
+JARNAME=`basename "$3"`
+DIRNAME=`cd "$DIRNAME">/dev/null; pwd`
+JOB_JAR_FILE="$DIRNAME/$JARNAME"
```

Comment:
Yes. BTW should the resourcemanager/jobtracker and namenode be mandatory? When the job is submitted from a cluster node I think these should not be set explicitly but if necessary the user could override them by the -jt and -fs toggles.